target/host.mk

```diff
@@ -39,3 +39,6 @@ ROOTFS_INSTALL_UNSTRIPPED := y
 # don't use sysroot on host
 LIBPHOENIX_DEVEL_MODE := n
 PREFIX_SYSROOT :=
+
+# check if linker is an apple linker
+APPLE_LD := $(shell $(LD) -v 2>&1 | grep -q Apple && echo 1 || echo 0)
```

Comment:
I assume you're sure that the `Apple` word won't appear if I would compile GNU binutils for Darwin)?

(there won't be `Apple MacOS X` instead of `Ubuntu` in gnu ld --version?)
![image](https://github.com/phoenix-rtos/phoenix-rtos-build/assets/3939983/b204d0b9-111b-445f-a4e1-af59e643dfb9).

Also please note that `$(LD)` is `gcc` here, not `ld` binary.